README.md

```diff
@@ -1,76 +1,71 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# LeaderBoard APIS Project
 
-> One paragraph statement about the project.
+> In this project I set up JavaScript for the Leaderboard list app, using webpack and ES6 features, notably modules. I develop a first working version of the app following a wireframe, but without styling - just focus on functionality. Then I consume the Leaderboard API using JavaScript async and await and add some styling.
 
 ![screenshot](./app_screenshot.png)
-
-Additional description about the project and its features.
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
+## Built With 🔨
+
+- JavaScript
+- HTML
+- CSS
+- Webpack
+- Visual Studio Code
+- Git & Github
 ## Live Demo
 
 [Live Demo Link](https://livedemo.com)
 
 
-## Getting Started
+### Prerequisites
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- IDE to edit and run the code (We use Visual Studio Code 🔥).
+- [Node.js](https://nodejs.org/en/download/) already downloaded.
+- Webpack - [click here to getting started](https://webpack.js.org/guides/getting-started/).
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
 ### Install
 
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
+To get a local copy up and running follow these simple example steps.
+- Open terminal
+- Clone this project by the command `git clone https://github.com/DanSam5K/LeaderBoard-APIS.git`
+- `cd <clone>` folder
+- Run `npm start` in your local browser or using Live Server in Visual Studio Code.
```

Comment:
- [ ] Please include the `npm install` instruction here.

Reply:
- Noted!

Reply:
This issue is well resolved! ✔️

Reply:
@rloterh Thank you 🤝